cardano-testnet/src/Testnet/Cardano.hs

```diff
@@ -1,7 +1,11 @@
 {-# LANGUAGE NamedFieldPuns #-}
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE TypeApplications #-}
+{-# OPTIONS_GHC -Wno-unused-do-bind #-}
+{-# OPTIONS_GHC -Wno-unused-matches #-}
+{-# OPTIONS_GHC -Wno-unused-top-binds #-}
```

Comment:
Why do we need `no-unused-matches` and `no-unused-top-binds`?

Reply:
The refactor uses Haskell variables for file resources.
The CLI wrapper functions return those resources.
However the refactor is still WIP some resources are defined but not yet used.
Once everything is don't the warnings should be turned on again.